minigrid/envs/babyai/goto.py

```diff
@@ -0,0 +1,276 @@
+"""
+Copied and adapted from https://github.com/mila-iqia/babyai.
+Levels described in the Baby AI ICLR 2019 submission, with the `Go to` instruction.
+"""
+from .core.levelgen import LevelGen, RejectSampling, RoomGridLevel
```

Comment:
Could these be updated to the full import statement not a local import.
Repeat this comment for all of the environments

Reply:
ok !